Test/Test/ConsoleReader.h

```diff
@@ -1,30 +1,13 @@
 #pragma once
-#include <iostream>
-#include <conio.h>
+
+const char esc = 27;
+const char enter = 13;
+const char backspace = 8;
 
 class ConsoleReader {
-	public:
+	private:
 		std::string buffer = "";
```

Comment:
There is no need to assign default value for string. I believe it initializes it with empty string on constructor. Anyway, as I mentioned, we shouldn't use empty string literal but empty string constant specific for OS.

Reply:
Thank you, fixed